roles/pulp/tasks/install.yml

```diff
@@ -102,12 +110,29 @@
         group: '{{ pulp_group }}'
         mode: 0775
 
-    - name: Create pulp install dir
+    # if it's already there, we don't want to change the owner
+    - name: Set permissions on pulp user home
       file:
-        path: '{{ pulp_install_dir }}'
+        path: '{{ pulp_user_home }}'
+        state: directory
+        mode: 0775
+      become: true
```

Comment:
Not sure if this is necessary, it's a leftover from trying some things -- I will remove it if I can (testing without it currently)